### PR TITLE
[git actions] fix unit test 

### PR DESCRIPTION
**Pull request by kongtiaowang**
fix unit test

### CODE REVIEW

**Comment by ridz1208** [comment]
@driusan do not merge before we discuss

**Comment by driusan** [comment]
Discuss what? Isn't this to fix the broken main branch?

File: test/unittests/NDB_BVL_Instrument_Test.php [hunk 1]
```diff
@@ -1549,7 +1549,7 @@ function testNullScores()
         $this->_setTableData();
         $this->_instrument->commentID = 'commentID1';
         $this->_instrument->table     = 'medical_history';
-        $this->_instrument->_nullScores(['Examiner' => 'Test Examiner1']);
+        $this->_instrument->_nullScores(['Test Examiner1'=> 'Examiner']);
```
Comment (ridz1208) [inline test/unittests/NDB_BVL_Instrument_Test.php:1552]:
```suggestion
        $this->_instrument->_nullScores(['Examiner']);
```

**Comment by ridz1208** [comment]
@kongtiaowang I made a little change on this PR (hopefully it will pass the tests) because the issue here was caused by another PR merged on 23 #7161 when that was merged and then I pushed the changes to main the test caught a valid error.

PR #7161 changed in a non-backwards compatible way the way the null scores function works and I made an issue a few days ago that we still havent solved #7421. Either way the change I made on this PR should work regardless of what happens with #7421

**Comment by ridz1208** [comment]
@driusan its passing

**Pull request merged**